benchmarks/dynamo/common.py

```diff
@@ -3379,7 +3403,20 @@ def get_example_inputs(self):
     run_mode_group.add_argument(
         "--inference", action="store_true", help="Performs inference"
     )
-    return parser.parse_args(args)
+
+    ret = parser.parse_args(args)
+    if "cuda" in ret.devices and "hpu" in ret.devices:
+        raise RuntimeError("Cannot use CUDA and HPU in one process due to GPU Migration usage. Please run them separately.")
+
+    new_dev_list = []
+    for d in ret.devices:
+        # GPU Migration will be applied depending on sys.argv contents
+        if d == "hpu":
+            new_dev_list.append("cuda")
```

Comment:
This means "hpu" will act as "cuda"?

Reply:
Yes, it uses a tool called gpu migration which intercepts cuda calls and executes Gaudi equivalents.